src/side-navigation/side-navigation.scss

```diff
@@ -1,205 +1,144 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../icons/index';
+@import '../icon/index';
 
 @mixin iui-side-navigation {
   display: inline-flex;
   flex: 0 0 auto;
   flex-direction: column;
-  overflow: hidden;
   transition: max-width $iui-speed-fast ease-out;
   height: 100%;
-
   @include themed {
-    background-color: t(iui-color-background-1);
-    border: 1px solid t(iui-color-background-5);
+    background-color: t(iui-color-background-3);
+    border-right: 1px solid t(iui-color-background-5);
   }
 
-  > .iui-expand-container {
-    height: $iui-l;
-    cursor: pointer;
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
-
-    > .iui-icon {
-      width: $iui-icons-small;
-      height: $iui-icons-small;
-
-      @include themed {
-        fill: t(iui-icons-color);
-      }
-    }
-
-    &:hover > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-primary);
-      }
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    &:focus-visible {
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
```

Comment:
This one is not needed. You can remove it and do this change:
![image](https://user-images.githubusercontent.com/36186912/118095513-d606a680-b3d8-11eb-8cff-261985a3d149.png)

Reply:
After making these changes the Administration button no longer sticks to the bottom of the side nav.
![Screen Shot 2021-05-13 at 11 00 22 AM](https://user-images.githubusercontent.com/849817/118144559-718c1d00-b3c1-11eb-9b43-87ce9cbe3e59.png)

Reply:
Pushed the change https://github.com/iTwin/iTwinUI/pull/73/commits/c842605757349eb53d5db75ba905b1fc99d4986d. Maybe you forgot to remove wrapper from HTML?